ElasticsearchTarget.php

```diff
@@ -68,7 +68,7 @@ class ElasticsearchTarget extends Target
     public function init()
     {
         parent::init();
-        $this->db = Instance::ensure($this->db, Connection::className());
+        $this->db = Instance::ensure($this->db, Connection::class);
```

Comment:
That would make it fail under older PHP versions.

Reply:
Missed that one. Thanks.